src/main/java/graphql/language/DirectiveDefinition.java

```diff
@@ -140,6 +153,7 @@ public DirectiveDefinition transform(Consumer<Builder> builderConsumer) {
         private SourceLocation sourceLocation;
         private List<Comment> comments = new ArrayList<>();
         private String name;
+        private boolean repeatable = false;
```

Comment:
it is false by default :-)

Reply:
just making it clear! I know java defaults but hey - a readder will see this and know as well